lib/rules/one-var.js

```diff
@@ -384,8 +384,13 @@ module.exports = {
             if (nodeIndex > 0) {
                 const previousNode = parent.body[nodeIndex - 1];
                 const isPreviousNodeDeclaration = previousNode.type === "VariableDeclaration";
+                const declarationsWithPrevious = declarations.concat(isPreviousNodeDeclaration && previousNode.declarations);
```

Comment:
nitpick: if can `isPreviousNodeDeclaration ` being `false`,  it could be pushed to `declarations`. can we change it to something like:
```js
const declarationsWithPrevious = declarations.concat(previousNode.declarations || []);
```